app/models/bus_stop.rb

```diff
@@ -13,6 +13,16 @@ class BusStop < ApplicationRecord
 
   scope :not_updated_since,
         ->(date) { where 'updated_at < ?', date.to_datetime }
+  validates :name, :hastus_id, :bench, :curb_cut, :lighting, :mounting,
+            :mounting_direction, :schedule_holder, :shelter, :sidewalk_width,
+            :trash, :bolt_on_base, :bus_pull_out_exists, :has_power,
+            :solar_lighting, :system_map_exists, :mounting_clearance,
+            :created_at, :updated_at, :sign_type, :shelter_condition,
+            :shelter_pad_condition, :shelter_pad_material, :shelter_type,
+            :shared_sign_post, :shelter_ada_compliance, :garage_responsible,
+            :bike_rack, :ada_landing_pad, :real_time_information, :state_road,
+            :need_work, :obstructions, :accessible, :stop_sticker,
+            :route_stickers, presence: true, if: :completed?
```

Comment:
I would put the `presence`, etc.. on a new line so that it stands out from the giant list.

Not a blocking change. This is temporary.

Reply:
```ruby
required_for_completion = %i[name hastus_id bench etc]

validates *required_for_completion, presence: true, if: :completed?
```

Reply:
Sure, could do. Saves some space, I suppose.

Reply:
Yeah, I mean, it's all code golf, really. But what I like is that it puts all of the stuff that `validates` is doing into one line.  It "`validates` the `presence` of all the fields which are `required_for_completion` `if` the stop is `completed`".

Reply:
Agreed. Separates the intent from the content.